earth_enterprise/rpms/opengee-fusion/pre-install.sh

```diff
@@ -15,13 +15,27 @@
 # limitations under the License.
 
 set +x
+NEW_INSTALL=false
+if [ "$1" = "1" ] ; then
+    NEW_INSTALL=true
+fi
 
 #-----------------------------------------------------------------
 # Main Functions
 #-----------------------------------------------------------------
 main_preinstall()
 {
+    # cleanup of special non-rpm upgrade case flag...
+    rm -f "$BASEINSTALLDIR_OPT/.upgrade-migration"
+
+    # Whether rpm or non-rpm upgrade...
     if [ -f /etc/init.d/gefusion ]; then
```

Comment:
Q: How did you test this code?  Were you able to do a non-RPM upgrade with these RPMs?

Reply:
This is based on the idea that you install the rpms without first removing the non-rpm install (that is, you do not run uninstall-server.sh or uninstall-fusion.sh first).  If you think this is not a valid scenario, then I could remove the migration checking entirely and maybe that would be a better way to address the concern you are stating.